pkg/genericapiserver/options/server_run_options.go

```diff
@@ -415,7 +415,7 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 	fs.MarkDeprecated("service-node-ports", "see --service-node-port-range instead")
 
 	fs.StringVar(&s.StorageConfig.Type, "storage-backend", s.StorageConfig.Type,
-		"The storage backend for persistence. Options: 'etcd2' (default), 'etcd3'.")
+		"The storage backend for persistence. Options: 'etcd2', 'etcd3' (default).")
```

Comment:
As long as I'm probably fine with changing the default for tests in this PR, I don't think we should do it for binaries (since we still don't have rollback etc., so we may not able to launch it).

@lavalamp - thoughts?